src/components/Club/club.utils.ts

```diff
@@ -392,6 +394,32 @@ export const useMutateClub = () => {
     },
   });
 
+  const depositClubFundsMutation = trpc.buzz.depositClubFunds.useMutation({
+    async onSuccess(_, { clubId }) {
+      await queryUtils.buzz.getBuzzAccount.invalidate({
+        accountId: clubId,
+        accountType: 'Club',
+      });
+      await queryUtils.buzz.getAccountTransactions.invalidate();
+    },
+    onError(error) {
+      try {
+        // If failed in the FE - TRPC error is a JSON string that contains an array of errors.
+        const parsedError = JSON.parse(error.message);
+        showErrorNotification({
+          title: 'Failed to deposit funds from club',
+          error: parsedError,
+        });
+      } catch (e) {
+        // Report old error as is:
+        showErrorNotification({
+          title: 'Failed to deposit funds from club',
+          error: new Error(error.message),
+        });
+      }
+    },
```

Comment:
Might be a good idea to have a utility method for this to avoid code duplication. No hard pressing tho